integration_test/agents/agents.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
why do we have licenses on some files but not others? (e.g. yaml files, bash scripts lack licences) Is there a policy you can point me to?

Reply:
We have this configuration for the `header-check` bot: 
https://github.com/GoogleCloudPlatform/ops-agent/blob/master/.github/header-checker-lint.yml

It doesn't run in this repo currently, but if it's there I assume we want it to at some point.

Reply:
before you do that for the bash scripts, let's find a more official policy. I'm having no luck finding any such policy on moma. I'm OK adding the licence to the go files for consistency with other go files.